packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -64,23 +68,27 @@ class _GestureArenaState {
 class GestureArena {
```

Comment:
If you're in here anyway, this seems like the class that should get a documentation boost :)